app/models/user.rb

```diff
@@ -206,6 +206,19 @@ def otp_verified?(otp)
     save!(validate: false)
   end
 
+  def lock!
```

Comment:
Consider renaming method, `#lock!` is provided by Active Record.

Reply:
Good point, any idea for new name?

Reply:
Also we can consider use term `block` instead of `lock`; rename script to `block_user` and use `block!` method to do that.

Reply:
I think `block` is the way to go.